velox/expression/fuzzer/tests/ArgGeneratorTestUtils.h

```diff
@@ -27,6 +27,13 @@ void assertReturnType(
     const exec::FunctionSignature& signature,
     const TypePtr& returnType);
 
+// Assert that the generated argument types meet user-specified check.
```

Comment:
Do you expect this function to be used in more than one test? If not, perhaps, move it to the only test that uses it.

Reply:
Moved this function to ArgGeneratorTest. Thanks.